com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1182,6 +1182,45 @@ def __init__(self, rest_client, credentials):
     def _get_url(self, req_name):
         return self._credentials['rest_url'] + self._credentials[req_name]
 
+    def submit_job(self, sab_file, application_name,  parameters=[], configuration={}, tracing='info' ):
+        """Submit a compiled Streams Application Bundle (a SAB file) to run in this Streamin Analytics service.
+        
+        Args:
+        sab_file (string): path to the compiled SAB file containing the application to be submitted
```

Comment:
The format of the args needs to be in Google style using Python types, e.g.

`sab_file(str)`

`configuration(dict)`  - no need to say it's optional. that's implied from the signature.

Reply:
Agreed -- I've changed the method accordingly.